googlemaps.py

```diff
@@ -0,0 +1,59 @@
+import requests
+
+
+def input_locations():
+    """
+    This allows the user to input the locations of the origin and destination and then puts these into the function to
+    get the distance
+
+    :return: inputted origins and destination
+    """
+    origin = input("Your location: ")
+    destination = input("Your destination: ")
+    return origin, destination
+
+
+def get_distance(origin, destination):
+    """
+    This function takes the inputted origin and destination, then uses the google maps API to calculate the distances
+    and duration (take out if not needed) of the route. These are outputted for each mode of transport so they are all
+    available.
+
+    :param origin: inputted origin
+    :param destination: inputted destination
+    :return: origin address, destination address and distances & durations of route for each mode of transport.
+    """
+    api_key = "Your API keys"
+    modes = ["driving", "walking", "bicycling", "transit"]
+    distances = dict()
+    durations = dict()
+    for mode in modes:
+        uri = f'https://maps.googleapis.com/maps/api/distancematrix/json?' \
+              f'origins={origin}&' \
+              f'destinations={destination}&' \
+              f'mode={mode}&' \
+              f'key={api_key}&language=en-GB'
```

Comment:
Maybe add a comment somewhere to remind person running script to add their API key in here.

Reply:
API keys are supposed to be added at line 25 - ` api_key = "Your API keys"`
I updated it to ` api_key = "Your API key here"` on the merged main branch on line 28 and hopefully it's clearer now